docs/blocks/administration/Billing.mdx

```diff
@@ -0,0 +1,513 @@
+import { Meta, Unstyled } from '@storybook/addon-docs';
+import { BillingFull } from './BillingFull' ;
+import { PayAndBilling } from './PayAndBilling' ;
+import { Tabs, TabPanel, TabList, Tab } from '../../../src/layout/Tabs';
+
+<Meta title="Blocks/Administration/Billing" />
+
+# Billing
+
+<Unstyled>
+  <Tabs direction="rtl">
+    <TabList>
+      <Tab className="light:text-white">
+        Demo
+      </Tab>
+      <Tab className="light:text-white">
+        Code
+      </Tab>
+    </TabList>
+    <TabPanel>
+      <div className="light:bg-white light:text-black flex flex-col border justify-center items-center border-gray-500 p-7">
+        <BillingFull />
+      </div>
+    </TabPanel>
+    <TabPanel>
+```tsx
+import React, { FC } from 'react';
+import { motion } from 'framer-motion';
+import {
+  Card,
+  Tab,
+  TabList,
+  Tabs,
+  TabPanel,
+  Stack,
+  Divider
+} from '../../../src/layout';
```

Comment:
should update these imports to be from `reablocks` so people can copy paste easier.

Reply:
right gotcha